modules/quantumdexBidAdapter.js

```diff
@@ -1,15 +1,25 @@
 import * as utils from '../src/utils.js';
 import { registerBidder } from '../src/adapters/bidderFactory.js';
 const BIDDER_CODE = 'quantumdex';
-const ENDPOINT = 'https://useast.quantumdex.io/auction/adapter';
-const USER_SYNC_URL = 'https://sync.quantumdex.io/usersync/adapter';
+const CONFIG = {
+  'quantumdex': {
+    'ENDPOINT': 'https://useast.quantumdex.io/auction/quantumdex',
+    'USERSYNC': 'https://sync.quantumdex.io/usersync/quantumdex'
+  },
+  'valueimpression': {
+    'ENDPOINT': 'https://useast.quantumdex.io/auction/adapter',
+    'USERSYNC': 'https://sync.quantumdex.io/usersync/adapter'
+  }
+};
+
+var bidderConfig = CONFIG['quantumdex'];
 var bySlotTargetKey = {};
 var bySlotSizesCount = {}
 
 export const spec = {
   code: BIDDER_CODE,
   supportedMediaTypes: ['banner', 'video'],
-  aliases: ['qde'],
+  aliases: ['valueimpression'],
```

Comment:
why the bidder code and default config are quantumdex but the short name is valueimpression?

Reply:
Hello,

Because we are merging the valueimpression adapter with the quantumdex adapter as described in https://github.com/prebid/Prebid.js/issues/5370, the valueimpression will be the alias of the quantumdex bid adapter. That's exactly what we want. Thanks

Reply:
ok then, thanks